### PR TITLE
User Guide updates to reflect latest version of Data Browser

### PR DESCRIPTION
**Pull request by brownhoward**
Restructured the user guide to have separate guides for each of the views. Sharing and Source have been populated, others are just placeholders and will need to be completed.

Updated the user guide to be consistent with the latest version of the Data Browser.

### CODE REVIEW

File: views/sharing/userguide.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+description: User Guide For Sharing Data Browser view
+---
+
+# Sharing View User Guide
+Solid allows you to control who can access folders and data resources within your Pod, and what permission have - read, create, update, delete.
```
Comment (TallTed) [inline views/sharing/userguide.md]:
```suggestion
Solid allows you to control who can access folders and data resources within your Pod, and what permission they have — i.e., read, create, update, and/or delete. 
```

File: views/sharing/userguide.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+<img src="Sharing_View.png" alt="Sharing" width="1024" style="border: 1; border-style:solid; border-color: rgb(200,200,200)">
+
+_Tip: Clicking the <img src="https://solid.github.io/solid-ui/src/originalIcons/go-to-this.png" alt="Sharing" width="16"> Goto This icon next to the user's Profile image/name will display the user's profile._
+
+## Specific Sharing for a Resource
+By default, resources within a folder inherit the permissions from the parent folder. 
+
+To set specific sharing for the resource:
+1. Click the **Set the sharing of folder contents separately from the sharing for the folder** button.
+2. This causes the a set of sharing groups (Owners, Editors, Posters, Submitters, Viewers) for the specific resource to be displayed.
```
Comment (TallTed) [inline views/sharing/userguide.md]:
```suggestion
2. This causes the sharing groups (Owners, Editors, Posters, Submitters, Viewers) for the specific resource to be displayed.
```

File: views/source/userguide.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+description: User Guide For Source Data Browser view
+---
+
+# Source View User Guide
+
+The sources for data resources (e.g., .ttl, html, etc.) can be viewed and edited via the Source view.
```
Comment (TallTed) [inline views/source/userguide.md]:
```suggestion
The source of data resources (e.g., Turtle, HTML, etc.) can be viewed and edited via the Source view. 
```

**Comment by brownhoward** [comment]
@TallTed Thanks for the updates. As we go through the process of updating the Data Browser, we will be attempting to keep the User Guide up-to-date. Hopefully the new structure for the guide will help with this.

File: README.md [hunk 4]
```diff
@@ -3,84 +3,208 @@ description: User Guide For Data Browser
...
+1. Open the dropdown menu in to top-right corner of the Data Browser by clicking your Profile image.
+2. Select **Log out**.
 
-## Chat
+## Pod Functionality
+Via the Data Browser, you can browse and maintain the data within the Pod.
 
-Creates a chat session
+### Navigation
+The Data Browser allows you to navigate through the data you have read access.
```
Comment (Vinnl) [inline README.md:48]:
```suggestion
The Data Browser allows you to navigate through the data you have read access to.
```

File: README.md [hunk 5]
```diff
@@ -3,84 +3,208 @@ description: User Guide For Data Browser
...
+1. Either click the Solid logo or select **Preferences** from the dropdown menu in the top-right of the Data Browser.
+2. Click the **Preferences** tab.
+
+#### Role
+To streamline the options presented when viewing data, you can select your role:
+* Normal User. Can view all standard social views.
+* Developer. In addition to Normal User, view Data, N3, and RDF views of the data.
+* Power User. In addition to Normal User, view additional Data Browser views such as Note Pad and Meetings.
+
+Multiple roles can be selected.
```
Comment (Vinnl) [inline README.md:151]:
```suggestion
Multiple roles can be selected by holding Shift.
```

File: views/sharing/userguide.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+## View
+To display the Sharing view for a folder or data resource:
+1. In the [Data Browser](https://github.com/solid/userguide/README.md), navigate to the resource.
+2. Select the <img src="https://solid.github.io/solid-ui/src/icons/padlock-timbl.svg" alt="Sharing" width="16" > Sharing view.
+3. The Sharing settings for the selected resource are displayed:
+
+<img src="Sharing_View.png" alt="Sharing" width="1024" style="border: 1; border-style:solid; border-color: rgb(200,200,200)">
+
+_Tip: Clicking the <img src="https://solid.github.io/solid-ui/src/originalIcons/go-to-this.png" alt="Sharing" width="16"> Goto This icon next to the user's Profile image/name will display the user's profile._
+
```
Comment (Vinnl) [inline views/sharing/userguide.md:17]: Not sure whether we're doing British or American English, but...

```suggestion

The Sharing view displays a table in which every row has its own colour, and represents a group of people. The first column holds the name of the group, the members of the group are in the second column, and the permissions they have is in the third.

```

**Review by Vinnl — APPROVED** [review]
This is very thorough, looks good!

**Pull request merged**